astropy/coordinates/tests/accuracy/test_fk4_no_e_fk4.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 
 from astropy import units as u
-from astropy.coordinates.angle_utilities import angular_separation
+from astropy.coordinates.angles import angular_separation
```

Comment:
Why don't we import public functions and classes from `astropy.coordinates`? There shouldn't be any import loops involving tests. I'm only highlighting it here, but this affects many test files.
```suggestion
from astropy.coordinates import angular_separation
```

Reply:
Agreed! I just did find-replace on these.

Reply:
DOne.